profile/Makefile

```diff
@@ -109,10 +109,17 @@ endif
 # intel Compiler
 ifeq ($(COMPILER),intel)
   CC = icpc
-  source += VectorizedSolver.cpp
+  #source += VectorizedSolver.cpp
   CFLAGS += -DINTEL
-  CFLAGS += -I/opt/intel/composer_xe_2013_sp1.3.174/mkl/include
-  LDFLAGS += -mkl
+  #CFLAGS += -I/your/path/to/mkl/include
+  #LDFLAGS += -mkl
+endif
+
+# MPI wrapped intel Compiler
+ifeq ($(COMPILER),impi)
+  CC = mpiicpc
+  CFLAGS += -DINTEL
```

Comment:
seems INTEL macro is used nowhere in the code. Rather, I find a ICPC macro when allocating aligned memory.

Reply:
fixed. It turns out memalign works with icpc